feat; document when http-metric timestamps happen (#1150)
grafana/k6-docs: src/data/markdown/translated-guides/en/02 Using k6/02 Metrics.md

These metrics are generated only when the test makes HTTP requests.

<Blockquote mod="note" title="">

For all `http_req_*` metrics, **the timestamp is emitted the end of the request.**

> In other words, the timestamp happens when k6 receives a return status, or the request times out.

Reviewer: ```suggestion
In other words, the timestamp happens when k6 receives the end of the response body, or the request times out.
```

It is not just the status. We do actually read the whole body before we emit the metrics
